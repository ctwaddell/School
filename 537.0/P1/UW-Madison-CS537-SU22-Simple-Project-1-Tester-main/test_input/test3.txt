a59
b40
q56
a5987
m78
o50
a11
A59!
ab59
a|5|9
