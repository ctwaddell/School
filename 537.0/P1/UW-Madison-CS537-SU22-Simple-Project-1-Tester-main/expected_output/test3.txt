a59
a5987
A59!
a|5|9
